bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/remote/ApiClientImpl.kt

```diff
@@ -0,0 +1,161 @@
+package io.bucketeer.sdk.android.internal.remote
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import io.bucketeer.sdk.android.internal.logd
+import io.bucketeer.sdk.android.internal.model.Event
+import io.bucketeer.sdk.android.internal.model.User
+import io.bucketeer.sdk.android.internal.model.request.GetEvaluationsRequest
+import io.bucketeer.sdk.android.internal.model.request.RegisterEventsRequest
+import io.bucketeer.sdk.android.internal.model.response.ErrorResponse
+import io.bucketeer.sdk.android.internal.model.response.GetEvaluationsResponse
+import io.bucketeer.sdk.android.internal.model.response.RegisterEventsResponse
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import java.util.concurrent.TimeUnit
+
+internal class ApiClientImpl(
+  endpoint: String,
+  private val apiKey: String,
+  private val featureTag: String,
+  private val moshi: Moshi
+) : ApiClient {
+
+  private val endpoint = endpoint.toHttpUrl()
+
+  private val client: OkHttpClient = OkHttpClient.Builder()
+    .build()
+
+  private val errorResponseJsonAdapter: JsonAdapter<ErrorResponse> by lazy {
+    moshi.adapter(ErrorResponse::class.java)
+  }
+
+  override fun fetchEvaluations(
```

Comment:
Could you rename the API name to `GetEvaluations`?
We use `fetchEvaluations` for the whole process, from getting the evaluations from the server to saving them into the DB.

Reply:
renamed to `ApiClient#getEvaluations`. [`c688c26` (#62)](https://github.com/ca-dp/bucketeer-android-sdk/pull/62/commits/c688c260f97503aa32d19f57e9393413b0cc7ead)